responsibleai/requirements-dev.txt

```diff
@@ -0,0 +1,30 @@
+# Requirements for raiwidgets development
+
+pytest==5.0.1
+mock==4.0.1
+pytest-mock==3.1.1
+requests==2.24.0
+
+autopep8==1.5.3
+flake8==3.8.4
+flake8-blind-except==0.1.1
+flake8-builtins==1.5.3
+flake8-copyright==0.2.2
+flake8-docstrings==1.5.0
+flake8-logging-format==0.6.0
+flake8-rst-docstrings==0.0.13
+requirements-parser==0.2.0
+
+wheel
+
+# Required for notebook tests
+nbformat
+papermill
+scrapbook
+jupyter
+nbval
+
+sphinx==3.1.1
+sphinx-gallery==0.8.1
+pydata-sphinx-theme==0.3.0
```

Comment:
Are we deploying this to a website? If not why are we building docs?

Reply:
I copied this file from raiwidget
removed.